modules/EmailMan/views/view.config.php

```diff
@@ -138,6 +138,7 @@ public function display()
         $configurator = new Configurator();
         $this->ss->assign('EMAIL_ENABLE_AUTO_SEND_OPT_IN', ($configurator->config['email_enable_auto_send_opt_in']) ? "checked='checked'" : "");
 
+
         $email_templates_arr = get_bean_select_array(true, 'EmailTemplate','name', '','name',true);
         $this->ss->assign('EMAIL_OPT_IN_TEMPLATES', get_select_options_with_id($email_templates_arr, $configurator->config['email_confirm_opt_in_email_template_id']));
```

Comment:
PHP Notice:  Undefined index: email_confirm_opt_in_email_template_id in /var/www/html/SuiteCRM/modules/EmailMan/views/view.config.php on line 143, referer: http://localhost/SuiteCRM/index.php?module=Administration&action=index

Reply:
done